pkgs/build-support/rust/default.nix

```diff
@@ -208,7 +209,7 @@ stdenv.mkDerivation (args // {
     ${stdenv.lib.optionalString (buildAndTestSubdir != null) "pushd ${buildAndTestSubdir}"}
     runHook preCheck
     echo "Running cargo test ${argstr} -- ''${checkFlags} ''${checkFlagsArray+''${checkFlagsArray[@]}}"
-    cargo test ${argstr} -- ''${checkFlags} ''${checkFlagsArray+"''${checkFlagsArray[@]}"}
+    cargo test ''${enableParallelBuilding:+-j $NIX_BUILD_CORES} ${argstr} -- ''${checkFlags} ''${checkFlagsArray+"''${checkFlagsArray[@]}"}
```

Comment:
We may want to set `test-threads` as well?

Reply:
Sounds reasonable, fixed. Thanks!